### PR TITLE
AB#38676 Cleaned up repo 

### PR DESCRIPTION
**Pull request by omersayshi**
### Overview

This repository was full of unnecessary/useless files and required a clean up.

### Testing Instructions
* Just run the app from the root of the repository 
* Make sure everything works as it's should

### CODE REVIEW

File: .fx/env.default.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "oauthHost": "https://login.microsoftonline.com",
+        "teamsWebAppId": "5e3ce6c0-2b1f-4285-8d4b-75ee78787346",
+        "oauthAuthority": "https://login.microsoftonline.com/8d87ccfa-0037-44f0-ac71-31c71ac81a2b",
+        "local_applicationIdUris": "api://localhost/da064752-5bc9-49db-b833-2095109e79ba"
+    },
+    "fx-resource-simple-auth": {
+        "filePath": "{{fx-resource-simple-auth.filePath}}",
+        "environmentVariableParams": "{{fx-resource-simple-auth.environmentVariableParams}}"
+    }
+}
```
Comment (gracet9182) [inline .fx/env.default.json]: Shouldn't be specific to your environment

File: .fx/settings.json [hunk 2]
```diff
@@ -14,5 +14,7 @@
             "fx-resource-aad-app-for-teams",
             "fx-resource-simple-auth"
         ]
-    }
+    },
+    "currentEnv": "default",
+    "projectId": "ada57d22-2e1d-4467-8ed1-16bcaf38e044"
```
Comment (gracet9182) [inline .fx/settings.json]: Can this be changed back to the original files?

**Review by lhao03 — APPROVED** [review]
Just pulled and ran, and it works! nice job! (●'◡'●) 

you uploaded your env.default.json, not the blank one. Delete it and replace with blank env.default.json and should be good

File: .fx/settings.json (hunk 2 above)
Comment (lhao03) [reply .fx/settings.json]: Yup this too, in addition to .fx/env.default.json

**Review by v-mschramm — APPROVED** [review]

**Review by gracet9182 — APPROVED** [review]

**Pull request merged**